examples/tfengine/team.hcl

```diff
@@ -348,26 +340,18 @@ template "project_apps" {
           name = "example-gke-subnet"
         }]
       }
-      # Add dependency on network deployment.
-      terraform_addons = {
-        deps = [{
-          name = "networks"
-          path = "../../example-prod-networks/resources"
-        }]
-      }
     }
     deployments = {
       resources = {
-        # TODO(user): Uncomment and re-run the engine after the apps project has been deployed.
-        # gke_clusters = [{
-        #   name                   = "example-gke-cluster"
-        #   network_project_id     = "example-prod-networks"
-        #   network                = "example-network"
-        #   subnet                 = "example-gke-subnet"
-        #   ip_range_pods_name     = "example-pods-range"
-        #   ip_range_services_name = "example-services-range"
-        #   master_ipv4_cidr_block = "192.168.0.0/28"
-        # }]
+        gke_clusters = [{
```

Comment:
does `plan` work? Or it's working now because the project and gke resources are in the same deployment?

Reply:
Right, terraform can handle references in data sources.